Content.Server/Climbing/ClimbSystem.cs

```diff
@@ -92,7 +92,7 @@ private void AddClimbableVerb(EntityUid uid, ClimbableComponent component, GetVe
         if (!args.CanAccess || !args.CanInteract || !_actionBlockerSystem.CanMove(args.User))
             return;
 
-        if (component.Bonk && _cfg.GetCVar(CCVars.GameTableBonk))
+        if (HasComp<BonkableComponent>(args.Target) && _cfg.GetCVar(CCVars.GameTableBonk))
             return;
```

Comment:
This isnt your fault but remove this check, it doesnt display the verb when bonking is allowed which makes no sense